### DISCUSSION TITLE
Problems running abp in Linux

### DISCUSSION THREAD

**Discussion by hgfernan** [abpframework/abp#9960] [Q&A]
Dear all, 

I'm trying to run ABP under Ubuntu Linux, but I'm facing problems with the ports in use: it seems that the browser tries to run in a port, but the server is listening to another one.

Has anybody seem that before ?

All the best,
Hilton

**Comment by abelsouzacosta** [comment]
I'm struggling on the same issue. After creating project, following the [tutorial](https://docs.abp.io/en/abp/latest/Getting-Started-Running-Solution?UI=MVC&DB=EF&Tiered=Yes) (Running the Solution) the IdentityServer is started on port 44335. The swagger runs on port 44348, but when i run the command `dotnet run` on server directory to start the application an Internal Server Error is always returned.
![Captura de tela de 2021-09-02 17-09-07](https://user-images.githubusercontent.com/30609244/131909326-35858448-4142-4a1c-971b-080acf29b6af.png)
![Captura de tela de 2021-09-02 17-09-36](https://user-images.githubusercontent.com/30609244/131909360-48be199c-b45e-4d32-9b46-51f8239e71df.png)
![Captura de tela de 2021-09-02 17-09-53](https://user-images.githubusercontent.com/30609244/131909394-f9212dd0-e39d-4f34-be5b-51440e2c110e.png)

**Comment by maliming** [comment]
hi

Please refer to the MS document
https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/?view=aspnetcore-5.0

**Comment by hgfernan** [comment]
Thanks for your answer, @maliming !

I think I'll have to browse **abp** sources to understand where it issues a `dotnet publish` command and change it to start the Web server that's available in my setup.
